toolkits/arcade_expert/arcade_arcade_expert/tools/search_documentation.py

```diff
@@ -0,0 +1,67 @@
+from typing import Annotated
+
+import httpx
+from arcade.sdk import ToolContext, tool
+from markdownify import markdownify
+from openai import OpenAI
+from openai.types.chat import ParsedChatCompletion
+
+from arcade_arcade_expert.models import Links
+
+
+@tool(requires_secrets=["OPENAI_API_KEY"])
```

Comment:
If we decide to have this in the prod worker, then we will need to add an `OPENAI_API_KEY` secret for all users